pkg/serverless/invocationlifecycle/lifecycle.go

```diff
@@ -69,7 +69,7 @@ func (lp *LifecycleProcessor) OnInvokeStart(startDetails *InvocationStartDetails
 
 	payloadBytes := []byte(lambdaPayloadString)
 	region, account, resource, arnParseErr := trigger.ParseArn(startDetails.InvokedFunctionARN)
-	if err != nil {
+	if arnParseErr != nil {
```

Comment:
We were checking the wrong error which meant if there was a problem parsing the function arn, no debug logs were created.